engine/table/src/main/java/io/deephaven/engine/table/impl/select/TimeSeriesFilter.java

```diff
@@ -107,7 +107,12 @@ public void setRecomputeListener(RecomputeListener listener) {
 
     @Override
     public TimeSeriesFilter copy() {
-        return new TimeSeriesFilter(columnName, nanos);
+        final TimeSeriesFilter copy = new TimeSeriesFilter(columnName, nanos);
+        if (initialized) {
+            copy.initialized = true;
+            UpdateGraphProcessor.DEFAULT.addSource(copy);
```

Comment:
This code as it currently exists is pretty buggy. We should not be in the UGP unless we have `setRecomputeListener` has been called. I think you should eliminate the `initialized` boolean, make `setRecomputeListener` check that the `listener` has not already been set, and do the UGP add there.